tools/wptrunner/wptrunner/tests/test_wpttest.py

```diff
@@ -48,3 +56,20 @@ def test_metadata_inherit():
     assert test_obj.min_assertion_count == 1
     assert test_obj.prefs == {"b": "c", "c": "d"}
     assert test_obj.tags == {"a", "dir:a"}
+
+@pytest.mark.skip(reason="See https://github.com/web-platform-tests/wpt/issues/12408")
```

Comment:
make this xfail rather than skip (we want it to pass, and we want to notice when it starts passing!)